hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/TestKeyValueHandlerWithUnhealthyContainer.java

```diff
@@ -177,7 +177,7 @@ public void testMarkContainerUnhealthyInFailedVolume() throws IOException {
 
   private KeyValueHandler getDummyHandler() {
     DatanodeDetails dnDetails = DatanodeDetails.newBuilder()
-        .setUuid(UUID.fromString(DATANODE_UUID))
+        .setID(DatanodeID.fromUuidString(DATANODE_UUID))
```

Comment:
Should we update `ContainerTestHelper#DATANODE_UUID` to be a `DatanodeID`?

Reply:
Sure, will fix it.